### DISCUSSION TITLE
Just to show appreciation

### DISCUSSION THREAD

**Discussion by shrekuntu** [gumyr/build123d#595] [General]
I just want to express my deep appreciation for all the hard work @gumyr and other contributors have put in.
Just when I though it can't be better than cadquery which is pretty good, you raised the bar with build123d. Excellent documentation helps a lot. Keep up a good work.

**Comment by gumyr** [comment]
Thank you!  It's good to know that users like build123d and are doing interesting things with it.

**Comment by bikemike** [comment]
I've just come across this project after taking a shallow dive into OpenSCAD. I can't believe how much more intuitive this is than the scripting language in OpenSCAD. I also love that it's BREP vs CSG and can export to many file formats!

Thanks to @gumyr and all the contributors for a really amazing tool! Everyone should know about this!

**Reply by gumyr** [reply]
Thank you so much for your kind words! We’re happy to hear that you find build123d intuitive and that the overall design resonates with you. It's feedback like yours that keeps us motivated to keep improving and expanding the tool.

If you’re interested in getting more involved, we’d love to have your input on future features or even contributions if you’re up for it! Whether it’s testing, documentation, or sharing your own projects made with build123d, there are many ways to help spread the word and make this tool even better.

Thanks again for your support, and we’re excited to see what you’ll create with build123d!
